APM/PSRCommandLogger.php

```diff
@@ -56,14 +61,24 @@ public function commandStarted(CommandStartedEvent $event)
             return;
         }
 
+        if ($this->stopwatch !== null) {
+            $this->stopwatch->start($event->getCommandName(), 'mongodb');
```

Comment:
I think this is preferable to the earlier suggestion about prefixing the name with `mongodb_`, as the category argument will ensure events are grouped (even if the default UI only displays text for the command name itself by default).

It's interesting that `stop()` doesn't also take a category name. That makes me wonder how it determines that we're closing the same event and whether we could possibly conflict if some other handler used a generic event name (e.g. "find", "update"); however, I haven't worked with the component directly so perhaps this isn't a concern.

Reply:
@jmikola the `stop` method always works on the last open section, not on a specific one. 🤷‍♂️